contrib/hashicorp/vault/vault_test.go

```diff
@@ -0,0 +1,440 @@
+package vault
+
+import (
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"net/http"
+	"net/http/httptest"
+	"testing"
+
+	"gopkg.in/DataDog/dd-trace-go.v1/ddtrace/ext"
+	"gopkg.in/DataDog/dd-trace-go.v1/ddtrace/mocktracer"
+
+	"github.com/hashicorp/vault/api"
+	"github.com/stretchr/testify/assert"
+)
+
+const secretMountPath = "/ns1/ns2/secret"
+
+func TestClient(t *testing.T) {
```

Comment:
I think we discussed in person but maybe we should also have it here: this test can be removed.